File: practice_java_program/notpad/TwoStringAnagram.java
import java.util.*;
public class TwoStringAnagram{
	public static void main(String[] args){
		String input1 = "listen";
		String input2 = "silent";
		if(input1.length() != input2.length()){
			System.out.println("both string are not anagram");
			return;
		}
		
		char[] ch1 = input1.toCharArray();
		char[] ch2 = input2.toCharArray();
		Arrays.sort(ch1);
		Arrays.sort(ch2);
		if(Arrays.equals(ch1,ch2)){
			System.out.println("both string are anagram");
		}
		else{
			System.out.println("both string are not anagram");
		}
	}
}

File: python_oriented_interview/notpad/TwoStringAnagram.py
class TwoStringAnagram:
    str1="silent"
    str2="listen"
    if len(str1)!=len(str2):
        print("this is not anagram")
    else:
        l1=list(str1)
        l2=list(str2)
        l1.sort()
        l2.sort()
        if l1==l2:
            print("this is anagram")
        else:
            print("this is not a anagram")